src/index.html

```diff
@@ -11,7 +11,123 @@
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">Full date of birth:
+          <input
+            type="date"
+            name="birth date"
+          >
+        </label>
+        <label class="form-field">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+            required
+          >
+        </label>
+        <label class="form-field">Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">Do you love cats?
+          <input
+            type="radio"
+            name="love_cats"
+            value="y"
+          >
+          Yes
+          <input
+            type="radio"
+            name="love_cats"
+            value="n"
+          >
+          No
+        </label>
+        <label class="form-field">What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="form-field">What time do you go to bed?
+          <input type="time" name="time_to_sleep">
+        </label>
+        <label class="form-field">What are your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">How do you rate our work?
+          <input
+            type="range"
+            name="rate_us"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="0"
+          >
+          </textarea>
+        </label>
```

Comment:
```suggestion
          ></textarea>
        </label>
```

для того, щоб textarea відпрацьовувала завжди спочатку, слід прибрати переноси та пробіли між відкриваючим та закриваючим тегами

https://user-images.githubusercontent.com/102413088/189630393-059bd50f-2033-4754-a5be-636f6a35e51f.mov